x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -80,25 +79,64 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
       .should('have.text', createdHost.hostname);
   });
 
+  it('should update endpoint policy on Endpoint', () => {
+    const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
+
+    loadPage(APP_ENDPOINTS_PATH);
+
+    cy.getByTestSubj('policyListRevNo')
+      .first()
```

Comment:
We have bad experience with `.first()` ;). With endpoint id stored we can make this selection explicit using `data-endpoint-id` attribute on table row. Then we can use it like 
```cy.get([data-endpoint-id=xxx]).within(() => cy.getByTestSUbj('policyNameCellLink').click())```